### PR TITLE
Add RSPEC_FILTER option to cp-test

### PR DESCRIPTION
**Pull request by alikins**
quoting cleanup based on shellcheck suggestions

### CODE REVIEW

File: docker/candlepin-base/cp-test [hunk 1]
```diff
@@ -123,6 +123,7 @@ while getopts ":dtrHulsc:p:v" opt; do
             RSPEC="1"
             DEPLOY="1"
             ;;
+        T  ) RSPEC_FILTER="${OPTARG}";;
```
Comment (awood) [inline docker/candlepin-base/cp-test]: Doesn't the `T` need to be added to the `getopts` command in the while loop?

**Comment by wottop** [comment]
buildbot: retest this please

**Comment by Ceiu** [comment]
Could we just modify -r to take the parameter or default to 1 if no filters were applied? I'd rather use cp-test -r "filter string" than -rT "filter string"

**Comment by cnsnyder** [comment]
Works well for me. ACK

**Pull request merged**